packages/core-utils/src/app/log.ts

```diff
@@ -1,16 +1,19 @@
 import debug from 'debug'
 import { Logger } from '../types'
 
+export const joinNewlinesAndDebug = (logs :string) =>
+  debug(logs.replace("\n", "\\n"))
+
```

Comment:
😍 

I might suggest a space before and after or a ` <\n> ` to make it stick out. I'm guessing that in many cases the newline will be the only space between text. I'm sure you've tested it, though, so happy to defer to your judgement!